### PR TITLE
separete Space into 3 things

### PR DESCRIPTION
**Pull request by Datseris**

### CODE REVIEW

**Comment by Datseris** [comment]
I've also reworked the docs ab it.

**Comment by Datseris** [comment]
https://juliadynamics.github.io/Agents.jl/previews/PR154

File: docs/src/index.md [hunk 1]
```diff
@@ -9,65 +9,43 @@ To get started, please read the [Tutorial](@ref) page.
...
+Implementing an ABM framework in Julia has several advantages:
+1. Using a general purpose programming language instead of a custom scripting language, such as NetLogo's, removes a learning step and provides a single environment for building the models and analyzing their results.
+2. Julia has a rich ecosystem for data analysis and visualization, implemented and maintained independently from Agents.jl.
+3. Julia is easier-to-use than Java (used for Repast and MASON), and provides a REPL (Read-Eval-Print-Loop) environment to build and analyze models interactively.
+4. Unlike Python (used for Mesa), Julia is fast to run. This is a crucial criterion for models that require considerable computations.
+5. Because the direct output of Agents.jl is a `DataFrame`, it makes it easy to use tools as DataVoyager.jl, which provide an [interactive environment](https://github.com/vega/voyager) to build custom plots from `DataFrame`s. (and of course the `DataFrame` itself is a tabular data format similar to Python's Pandas).
```
Comment (kavir1698) [inline docs/src/index.md]: use tools, such as ...

File: docs/src/tutorial.md [hunk 2]
```diff
@@ -26,14 +26,18 @@ AgentBasedModel
...
 
-Regardless, in every case you initialize your space by calling `Space`:
+```@docs
+GraphSpace
+GridSpace
+```
 
+and the continuous version is
 ```@docs
-Space
```
Comment (kavir1698) [inline docs/src/tutorial.md]: In the docs, it is still showing `Space`, rather than `ContinuousSpace`.

**Comment by kavir1698** [comment]
I would remove features from README so that we don't have to maintain two list of features, one in README and one in the docs.

**Pull request merged**